api/src/main/java/org/openmrs/module/eptsreports/metadata/CommonMetadata.java

```diff
@@ -38,6 +38,8 @@
 	
 	private String RETURN_VISIT_DATE = "e1dae630-1d5f-11e0-b929-000c29ad1d07";
 	
+	private String TRANSFER_FROM_OTHER_FACILITY = "e1da7d3a-1d5f-11e0-b929-000c29ad1d07";
```

Comment:
Where is this used @rubailly ?

Reply:
[Here](https://github.com/esaude/openmrs-module-eptsreports/blob/TxNew-TxCurr-report-queries-fix/api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/SqlCohortQueries.java#L294), in the #hivMetadata.getTransferFromOtherFacilityConcept() method. The specs says we want patients for whom the answer to the  question "ARV Plan" is "Start drugs" **OR** "Transfer from other facility"

Reply:
See it, not sure how I missed it before.

Thanks